### PR TITLE
provide Parallel() to run test parallel

### PR DESCRIPTION
**Pull request by tiancaiamao**

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @iamxy

File: check.go [hunk 1]
```diff
@@ -611,13 +612,23 @@ func (runner *suiteRunner) run() *Result {
...
+					case <-c.done:
+					case <-c.parallel:
+						delayedC = append(delayedC, c)
+					}
 					if c.status() == fixturePanickedSt {
 						runner.skipTests(missedSt, runner.tests[i+1:])
 						break
 					}
 				}
+				// wait those parallel tests finish.
```
Comment (zimulala) [inline check.go]: s/wait/Wait

**Comment by zimulala** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Comment by Deardrops** [comment]
3 years later, I find out that `c.Parallel()` don't take effort finally.

**Pull request merged**